app/src/main/res/layout/fragment_marker.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    >
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.framgia.fdms.screen.marker.MarkerViewModel"
+            />
+
+    </data>
+    <LinearLayout
```

Comment:
WARNING: [UselessLeaf] This `LinearLayout` view is useless (no children, no `background`, no `id`, no `style`)
 A layout that has no children or no background can often be removed (since it is invisible) for a flatter and more efficient layout hierarchy.